src/qmflows/parsers/cp2KParser.py

```diff
@@ -290,16 +315,151 @@ def readCp2KBasis(path: str) -> tuple:
     return (basiskey, basisData)
 
 
+#: A :class:`~collections.abc.Sequence` typevar.
+ST = TypeVar('ST', bound=Sequence)
+
+
+@overload
```

Comment:
I don't understand this overload :) . What is Literal_[1] doing?

Reply:
[``Literal``](https://docs.python.org/3/library/typing.html#typing.Literal) is, as the name implies, a literal ``1``.
In other words, if ``n == 1`` do this and if ``isinstance(n, int) and n != 1`` do that.

Reply:
now I got it